src/addrman.cpp

```diff
@@ -573,6 +571,7 @@ void CAddrMan::GetAddr_(std::vector<CAddress>& vAddr, size_t max_addresses, size
 
         vAddr.push_back(ai);
     }
+    LogPrint(BCLog::ADDRMAN, "Created getaddr response of size %d\n", vAddr.size());
```

Comment:
nits:
1. Say "of size XYZ entries" (implying not bytes).
2. Maybe something more specific like "Selected XYZ entries to be sent in response to GETADDR"

Reply:
Thanks - I forgot to answer this - will address it when I push next (i.e. after #22950 is merged and I'll need to rebase anyway).

Reply:
Why so pessimistic? :)
This already has 3 ACKs, so might bring this up at the meeting? Unless you explicitly want to avoid forcing a rebase on #22950

Reply:
I think that since #22950 is so much larger and requires more time-consuming review, it makes sense to try to merge that first and rebase other PRs on top of it (rather than merge small PRs and force 22950 to be rebased/re-reviewed).

Reply:
Yes, I agree with that. I'd much rather rebase this than go through all the (mostly mechanical) changes of 22950 another time.

Reply:
I took your second suggestion, thanks!